src/main.cpp

```diff
@@ -0,0 +1,16 @@
+#include <ethernetModule.h>
+#include <headerConfig.h>
+#include <Arduino.h>
+
+char defaultMessage[] = "Default Message"; 
+// This can be any wanted character array / string
+
+int main() {
+  eth::setup();
+
+  while(1) {
+    eth::read();
+    eth::write(defaultMessage);
+    delay(1000);
```

Comment:
If you want same behavior as the `ethernetCheck()` function, this probably should look something like:
```suggestion
    char* receivedMessage = eth::read();
    if (receivedMessage == nullptr) {
        eth::write(defaultMessage);
    }
    delay(1000);
```
But it's fine for me to leave it as is for now.